gold/core_test.cpp

```diff
@@ -0,0 +1,62 @@
+
+#include "Gold_core.hpp"
+
+
+uint8_t get_byte(uint64_t addr) {
+  return addr>>4;
+}
+
+int main() {
```

Comment:
This would be a good place to explain what the Golden model does and how it works.  It appears that `oldest_ld` is an pending load instruction and line 20 means that the load address is 0x135?  line 28 seems to mean that the load copies the data from core[0x135] to oldest_ld[0x135] (abusing notation a bit).
What is `bad_spec_ld` supposed to illustrate?  It seems to do the same for a 3 byte range.